https://chromium-review.googlesource.com/c/chromium/src/+/4545069

Parent:     4d0ef938 ([DTC] Replace Browser Test Management Logic with new Mixin)
Author:     Stephan Hartmann <stha09@googlemail.com>
AuthorDate: 2023-05-30 20:27:03 +0000
Commit:     Chromium LUCI CQ <chromium-scoped@luci-project-accounts.iam.gserviceaccount.com>
CommitDate: 2023-05-30 20:27:03 +0000
IWYU: add cmath for various math functions in //skia
Bug: 957519
Change-Id: I4616f7f161330feb3fb2299da094874663655189
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/4545069
Commit-Queue: Nico Weber <thakis@chromium.org>
Reviewed-by: Nico Weber <thakis@chromium.org>
Cr-Commit-Position: refs/heads/main@{#1150817}
diff --git a/skia/ext/skcolorspace_trfn.cc b/skia/ext/skcolorspace_trfn.cc
index c0d6bd2..97c4dfa 100644
--- a/skia/ext/skcolorspace_trfn.cc
+++ b/skia/ext/skcolorspace_trfn.cc
@@ -4,6 +4,8 @@
 
 #include "skia/ext/skcolorspace_trfn.h"
 
+#include <cmath>
+
 namespace skia {
 
 namespace {
